src/common/Cargo.toml

```diff
@@ -23,6 +23,7 @@ lru = { git = "https://github.com/singularity-data/lru-rs.git", rev = "e0e9ddaf8
 madsim = "=0.2.0-alpha.3"
 memcomparable = { path = "../utils/memcomparable" }
 more-asserts = "0.3"
+mysql_async = "0.30"
```

Comment:
Don't add more things to `common` crate 🤣

Reply:
+1